include/api.php

```diff
@@ -1923,7 +1924,7 @@ function api_favorites_create_destroy($type){
 				$itemid, api_user());
 
 		if ($item===false || count($item)==0)
```

Comment:
Standards: Please add braces to this conditional statement and use `dbm::is_result()` instead.